voluncheer/templates/profiles/profile.html

```diff
@@ -210,6 +214,18 @@ <h2>MY BADGES</h2>
                 <p class="hours-required"> You have {{ hours_required }} hours to go until your next badge!</p>
             {% endif %}
         </div>
+
+        <h2>MY GALLERY</h2>
+        {% for post in gallery_post %}
+            <article class="gallery-post">
+                {% if post.photo %}
+                <a href="#"><img src="{{post.photo.url}}" alt="" class="gallery-post-image" width="220px"/></a>
+                {% endif %}
+                <p class="gallery-post-disciption-h1">{{ post.title }}</p>
```

Comment:
discription should be spelled "description"

Reply:
Fixed